src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -49,12 +49,13 @@ struct SemidiscretizationHyperbolicParabolic{Mesh, Equations, EquationsParabolic
 
     new(mesh, equations, equations_parabolic, initial_condition,
         boundary_conditions, boundary_conditions_parabolic,
-        source_terms, solver, cache, cache_parabolic, performance_counter)
+        source_terms, solver, solver_parabolic, cache, cache_parabolic, performance_counter)
   end
 end
 
 """
     SemidiscretizationHyperbolicParabolic(mesh, both_equations, initial_condition, solver;
+                                          solver_parabolic=default_parabolic_solver(),
```

Comment:
Maybe also something for the list: Do we want to let the `default_parabolic_solver` depend on, say, `both_equations` and/or the `solver`?

Reply:
I see the flexibility this would bring, but do we already have a concrete use case where this would be necessary/interesting in the near future? If not, I'd keep it simple for now and extend it later ("make it work" first). But I wouldn't object to doing it now if it is necessary/useful.